lib/DirectoryWatcher.js

```diff
@@ -51,7 +51,7 @@ function DirectoryWatcher(directoryPath, options) {
 	this.watcher = chokidar.watch(directoryPath, {
 		ignoreInitial: true,
 		persistent: true,
-		followSymlinks: false,
+		followSymlinks: !!options.followSymlinks,
```

Comment:
Breaking change, default is not false, default is boolean value `options.followSymlinks`

Reply:
If you don't specify `options.followSymlinks`, it will be `undefined`. Well, we'll have `!!options.followSymlinks === !!undefined === false`. So yes, default is still false, no breaking change.

Reply:
@birdy- hm, you are right

Reply:
We need it !